### PR TITLE
Run chaos plan

### PR DESCRIPTION
**Pull request by panda-sheep**
As title

### CODE REVIEW

File: mock/nebula/etc/nebula-storaged.conf [hunk 1]
```diff
@@ -15,68 +15,80 @@
...
+--engine_type=rocksdb
 
 # Compression algorithm, options: no,snappy,lz4,lz4hc,zlib,bzip2,zstd
 # For the sake of binary compatibility, the default value is snappy.
 # Recommend to use:
 #   * lz4 to gain more CPU performance, with the same compression ratio with snappy
 #   * zstd to occupy less disk space
 #   * lz4hc for the read-heavy write-light scenario
---rocksdb_compression=snappy
+--rocksdb_compression=lz4
```
Comment (bright-starry-sky) [inline mock/nebula/etc/nebula-storaged.conf:60]: For what purpose?

File: src/nebula/client/GraphClient.cpp [hunk 2]
```diff
@@ -52,18 +52,22 @@ ErrorCode GraphClient::execute(folly::StringPiece stmt,
...
                 return exeRet.errorCode();
             }
             sleep(retry);
             continue;
         }
-        resp = *(const_cast<nebula::DataSet*>(exeRet.data()));
+
+        // Not every ResultSet returned by Session::execute contains a DataSet
+        if (exeRet.data()) {
+            resp = *(const_cast<nebula::DataSet*>(exeRet.data()));
```
Comment (critical27) [inline src/nebula/client/GraphClient.cpp:69]: If not data, should we return SUCCEEDED. BTW, if `exeRet` has the message, maybe we need them as well. Some action need it as well.

**Review by critical27 — COMMENTED** [review]
Generally LGTM

File: mock/nebula/etc/nebula-storaged.conf (hunk 1 above)
Comment (panda-sheep) [reply mock/nebula/etc/nebula-storaged.conf:60]: This is the newest config file in nebula 2.0.

**Review by bright-starry-sky — APPROVED** [review]

**Pull request merged**